utility/firebase.js

```diff
@@ -571,7 +571,10 @@ export const getAllApplicants = async callback => {
     .where('status.applicationStatus', '!=', 'inProgress')
     .onSnapshot(snap => {
       callback(
-        snap.docs.map(doc => doc.data()).sort((a, b) => a.basicInfo.firstName.localeCompare(b.basicInfo.firstName))
+        snap.docs
+          .map(doc => doc.data())
+          .filter(a => a.basicInfo.identifyAsUnderrepresented !== 'no') // cmd-f filter; remove after
+          .sort((a, b) => a.basicInfo.legalFirstName.localeCompare(b.basicInfo.legalFirstName))
```

Comment:
We might need a stronger/alternative sort here (like also add last name as secondary sort) since we ran into problems with nwHacks where some applicant placements kept jumping if they had the same first name before/after they were graded.

Reply:
ok added last name sort